### PR TITLE
Add travis-ci support to build new tags for Dockerhub

### PR DESCRIPTION
**Pull request by rwinfosec**

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+version=v$(date +'%m/%d/%Y')
```
Comment (bpholt) [inline deploy.sh]: Let's use `YYYY-MM-DD` instead

**Review by bpholt — CHANGES_REQUESTED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+env:
+  global:
+    secure: gkDTNI2TORFDqE8MgF8TnRGBaAP8IeJiWQr6O/gC4JfhzxZrwGlAefh6KRdd21YeJRjIgLJCxLd9rFLsCTiOK9PgAMic2IycnbJIueXfLgJpixhbveXAUhovxhZnAakSGaC6grbAMJkSTs3KtIe5zkDEOiPTxoDl3BnjSyLyK536UghtsRr2eaJpeFT7wiDfssPA39DDk9G5wC+xn3YadN+/zRw1lWzegJCCAJHDGGf85LvS9MxhvXa8safEpEEG1T6DZB7oc0c33J0tiAs5QKajlawm3yI+NFl4C4uBlasW4aEUewTuQEKqd5vy27JM3mcN7qO3uEhV4oPL29iVaC8iIRJ1exItoEgzoQQROqjHu93WILnpwhLqXPMmbickZLCngmtgwbSoRlaQ5ov+3ZGiz/7fTGHAWA3tQv39Nv99DGbYsMIYea9f4SqnjJjcO+dwYkT3jr2Vylb5cNV2O64SMt4BPU29z0btB+qrSMM9Ki7LbO2tv/0yUzmFdHTk1oDKF+7NRrRG7HFeFkAGzqHVD8Fd1OMiHsBrRr5itBlfi5olYEI+MEyhPkWY7lK94F26AMFV67dg4B+OaxPSPAUsp58sUp8Swi5oWF17Cs/23gagR16h07ozkeAhIw5+kGmWbAP+5Hp4H/FiVdkC/eYXShCuTC4qNrxfFEmjx6Y=
+branches:
+  only:
+    - master
```
Comment (bpholt) [inline .travis.yml]: This might be a good time to change this repo to use `main` instead of `master`.

Comment (rwinfosec) [reply .travis.yml]: Updated to use main and re-targeted PR

File: deploy.sh (hunk 1 above)
Comment (rwinfosec) [reply deploy.sh]: Updated date format to be vYYYY-MM-DD

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  global:
+    secure: gkDTNI2TORFDqE8MgF8TnRGBaAP8IeJiWQr6O/gC4JfhzxZrwGlAefh6KRdd21YeJRjIgLJCxLd9rFLsCTiOK9PgAMic2IycnbJIueXfLgJpixhbveXAUhovxhZnAakSGaC6grbAMJkSTs3KtIe5zkDEOiPTxoDl3BnjSyLyK536UghtsRr2eaJpeFT7wiDfssPA39DDk9G5wC+xn3YadN+/zRw1lWzegJCCAJHDGGf85LvS9MxhvXa8safEpEEG1T6DZB7oc0c33J0tiAs5QKajlawm3yI+NFl4C4uBlasW4aEUewTuQEKqd5vy27JM3mcN7qO3uEhV4oPL29iVaC8iIRJ1exItoEgzoQQROqjHu93WILnpwhLqXPMmbickZLCngmtgwbSoRlaQ5ov+3ZGiz/7fTGHAWA3tQv39Nv99DGbYsMIYea9f4SqnjJjcO+dwYkT3jr2Vylb5cNV2O64SMt4BPU29z0btB+qrSMM9Ki7LbO2tv/0yUzmFdHTk1oDKF+7NRrRG7HFeFkAGzqHVD8Fd1OMiHsBrRr5itBlfi5olYEI+MEyhPkWY7lK94F26AMFV67dg4B+OaxPSPAUsp58sUp8Swi5oWF17Cs/23gagR16h07ozkeAhIw5+kGmWbAP+5Hp4H/FiVdkC/eYXShCuTC4qNrxfFEmjx6Y=
+branches:
+  only:
+    - main
+jobs:
+  include:
+      - stage: deploy
+        script:
+          - if [[ ($TRAVIS_EVENT_TYPE == "push" || $TRAVIS_EVENT_TYPE = "cron") && $TRAVIS_BRANCH == "main" ]] ; then ./deploy ; fi
```
Comment (rwinfosec) [inline .travis.yml:11]: This will only run if it is a cron job or merge to main

**Review by charlesdunbar — APPROVED** [review]

**Pull request merged**